third_party/terraform/website/docs/d/datasource_tpu_tensorflow_versions.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "google"
+page_title: "Google: google_tpu_tensorflow_versions"
+sidebar_current: "docs-google-datasource-tpu-tensorflow-versions"
+description: |-
+  Get available TensorFlow versions.
+---
+
+# google\_tpu\_tensorflow\_versions
+
+Get TensorFlow versions available for a project. For more information see the [official documentation](https://cloud.google.com/tpu/docs/) and [API](https://cloud.google.com/tpu/docs/reference/rest/v1/projects.locations.tensorflowVersions).
+
+## Example Usage
+
+```hcl
+data "google_tpu_tensorflow_versions" "available" { }
+```
+
+## Example Usage: Configure Basic TPU Node with available version
+
+```hcl
+data "google_tpu_tensorflow_versions" "available" { }
+
+resource "google_tpu_node" "tpu" {
+	name           = "test-tpu"
+	zone           = "us-central1-b"
+
+	accelerator_type   = "v3-8"
+	tensorflow_version = "${data.google_tpu_tensorflow_versions.available.versions[0]}"
+	cidr_block         = "10.2.0.0/29"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - (Optional) The project to list versions for. If it
+    is not provided, the provider project is used.
+
+* `region` - (Optional) The region to list versions for. If it
```

Comment:
This should be `zone`, right? Based on [this](https://github.com/terraform-providers/terraform-provider-google/blob/master/google/data_source_tpu_tensorflow_versions.go#L21-L25) / the comment on `versions`.